### ISSUE TITLE
№264 Для наблюдателей за статусами при уходе из этого статуса также отправлять уведомление

### ISSUE THREAD

**Issue by BlizD** [BlizD/Tasks#72]
№264 Для наблюдателей за статусами при уходе из этого статуса также отправлять уведомление

**Comment by capitoshko** [comment]
А есть смысл? Я очевидно в разомном состоянии не хочу следить за задачей. Зачем меня еще спамить на почту в телеграм?

**Comment by BlizD** [comment]
@capitoshko пока наверно не сильно требуется. Но это важно когда используется методология Канбан и WIP лимиты в ней. Тогда ответственному за статус, важно знать когда появилось свободное место в его колонке. 
Например: заказчик увидел, что из статуса "Задачи к разработке" была взята задача в работу. В этом случае ему необходимо подумать, какую следующую задачу перенести в этот статус.

**Comment by capitoshko** [comment]
> @capitoshko пока наверно не сильно требуется. Но это важно когда используется методология Канбан и WIP лимиты в ней. Тогда ответственному за статус, важно знать когда появилось свободное место в его колонке.
> Например: заказчик увидел, что из статуса "Задачи к разработке" была взята задача в работу. В этом случае ему необходимо подумать, какую следующую задачу перенести в этот статус.

Согласен. В таком случае это удобная штука.

**Comment by 7OH** [comment]
Добавляем галку "Следить за уходом" ? Можно попробовать.

**Comment by capitoshko** [comment]
Думаю, что нужно добавить механизм в отслеживание, а не делать ещё одну команду. В матрицу отслеживания,и которую Антон выложил, добавить ещё одно событие

**Comment by BlizD** [comment]
Да, тут надо смотреть в сторону доработки механизма уведомлений.

**Comment by BlizD** [comment]
Пока без изменения метаданных
